hudi-common/src/main/java/org/apache/hudi/common/io/storage/SizeAwareFSDataOutputStream.java

```diff
@@ -43,7 +43,7 @@
 
   public SizeAwareFSDataOutputStream(Path path, FSDataOutputStream out, ConsistencyGuard consistencyGuard,
       Runnable closeCallback) throws IOException {
-    super(out);
+    super(out, null);
```

Comment:
is it okay to pass null here? this is the stats object right?

Reply:
> is it okay to pass null here? this is the stats object right?

Actually, `super(out)` is equivalent to `super(out, null)`. Source code below:
```
  @Deprecated
  public FSDataOutputStream(OutputStream out) throws IOException {
    this(out, null);
  }
```